views/index.handlebars

```diff
@@ -31,5 +31,32 @@
         <p>There are no items yet. Click "List New Item" to add one.</p>
     {{/if}}
 {{else}}
-    <p>This is the about page.</p>
+    <hr>
+    <p><strong>What is Unisell?</strong></p>
+    <p>Unisell is a used-item marketplace exclusive to university students. Verified university students can buy or sell used 
+    items.</p>
+    <hr>
+    <p><strong>Our goal</strong></p>
+    <p>Unisell is designed to improve college communities and promote reuse of items by providing a platform where students 
+    can safely interact.</p>
+    <hr>
+    <p><strong>How does Unisell work?</strong></p>
+    <p>To join us, all you need is your student email address.
+        Once you create an account, you can post or buy items. 
+    </p>
+        <p>Purchasing items:</p>
+        <ol>
+            <li>Leave a comment on the post you wish to purchase if you have any questions.</li>
+            <li>Create a bid.</li>
+            <li>You will be notified if your bid is accepted.</li>
+        </ol>
+        <p>Selling items:</p>
+        <ol>
+            <li>Create a post with photos, keywords, descriptions, etc.</li>
+            <li>Accept a bid that you like.</li>
+            <li>Communicate with the buyer to decide when and where to meet.</li>
+        </ol>        
+        <p><strong>*Note that all transactions happen on campus.</strong></p>
+
+</p>
```

Comment:
@gpark31 we also need to list which universities are on unisell (like in a ul). Take a look at the auth pages to see how to grab universities